core/runner/src/main/java/com/cognifide/aet/runner/configuration/RunnerConfigurationConf.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.runner.configuration;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "AET Runner configuration", description = "AET Runner configuration")
+public @interface RunnerConfigurationConf {
+
+  int DEFAULT_TASK_RUN_FAILURE_TIMEOUT_SECONDS = 120;
+
+  String PARAM_FAILURE_TIMEOUT = "ft";
```

Comment:
This is used as a label now - please change to "Failure timeout"

Reply:
Done ;)